### PR TITLE
Resizing behaviour

### PR DESCRIPTION
**Pull request by npriyadarshi**
### Platforms Impacted
- [x] iOS
- [ ] macOS

### Description of changes

DrawerController.swift 
- on changing the resizingBehaviour, we check if the drawer is loaded, and accordingly, update the resizingHandleView(as per the provided resizingBehaviour)

DrawerDemoController.swift
- Added new controls in content view to toggle resizing behaviour of a presented drawer controller
- Created new buttons with resizing controls for top and bottom drawer.

### Verification
- Tested the FluentUI.demo app manually. Ensured that, upon changing the resizing behaviour, the changes are reflected in the layout.

Tested the existing drawer controllers to ensure that no behaviour is regressed.
| Before                                       | After                                      |
|----------------------------------------------|--------------------------------------------|
| Changing resizing behaviour of a presented drawer, did not update the layout to add or remove the resizingHandleView | We can see that the changes are appropriately reflected, i.e. when behaviour is changed from .none to any other enum, resizingHandleView should appear and when behaviour is changed from any to .none, the resizingHandleView should disappear|

### Pull request checklist

This PR has considered:
- [x] Light and Dark appearances
- [x] VoiceOver and Keyboard Accessibility
- [x] Internationalization and Right to Left layouts
- [x] Different resolutions (1x, 2x, 3x)
- [x] Size classes and window sizes (iPhone vs iPad, notched devices, multitasking, different window sizes, etc)

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/fluentui-apple/pull/266)

### CODE REVIEW

File: ios/FluentUI/Drawer/DrawerController.swift [hunk 1]
```diff
@@ -253,6 +253,11 @@ open class DrawerController: UIViewController {
             if presentationDirection.isHorizontal && resizingBehavior == .dismissOrExpand {
                 resizingBehavior = .dismiss
             }
+            
+            // If the drawer is already loaded, update the resizing handle when resizing behaviour is changed.
+            if self.isViewLoaded {
```
Comment (harrieshin) [inline ios/FluentUI/Drawer/DrawerController.swift:258]: shouldn't need self. also applies to other callers

File: ios/FluentUI/Drawer/DrawerController.swift [hunk 2]
```diff
@@ -675,6 +666,7 @@ open class DrawerController: UIViewController {
                 initResizingHandleView()
                 if presentationDirection == .down {
                     containerView.addArrangedSubview(newView)
+                    containerView.layoutIfNeeded()
```
Comment (harrieshin) [inline ios/FluentUI/Drawer/DrawerController.swift:669]: why are we updating layout only if statement?

Comment (npriyadarshi) [reply ios/FluentUI/Drawer/DrawerController.swift:669]: When the presentationDirection == .down, i could observe that, when .none is changed to any other resizing behaviour, the resizingHandleView is added to the containVeiw UIStack with a wired animation of the resizingHandleView coming from the top of the screen. layoutIfNeeded immediately does the pending layout and the animation is avoided.

File: ios/FluentUI/Drawer/DrawerController.swift (hunk 1 above)
Comment (npriyadarshi) [reply ios/FluentUI/Drawer/DrawerController.swift:258]: i will update this.

File: ios/FluentUI/Drawer/DrawerController.swift (hunk 2 above)
Comment (npriyadarshi) [reply ios/FluentUI/Drawer/DrawerController.swift:672]: updated the containerView.layoutIfNeeded() instruction with a comment to denote its purpose.

**Review by harrieshin — APPROVED** [review]

**Pull request merged**